docs/docs/org-management/workspaces/workspace-variables.md

```diff
@@ -3,6 +3,12 @@ id: workspace-variables
 title: Workspace Variables
 ---
 
+:::caution
+Workspace variable feature is deprecated and will be removed in the future releases. In the current version, you can only delete existing variables; creating and updating variables through this deprecated feature is not possible.
```

Comment:
```suggestion
Workspace variables are currently marked as deprecated, indicating that it will be removed in future releases. In the current version, you are still able to delete existing variables and use it through out any ToolJet apps, but creating and updating variables are no longer supported. 
```